vanilla-tests/src/main/java/fixtures/bodyboolean/Bools.java

```diff
@@ -141,7 +141,8 @@
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public boolean getTrue() {
-        return getTrueAsync().block();
+        Boolean value = getTrueAsync().block();
+        return value == null ? false : value;
```

Comment:
Currently I return default value for the primitive type, if result is `null`.

Or should we throw certain exception?